cmd/query/app/qualitymetrics/data_test.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package qualitymetrics
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/jaegertracing/jaeger/pkg/testutils"
+)
+
+func TestGetSampleData(t *testing.T) {
+	data := GetSampleData()
+	assert.NotNil(t, data.TraceQualityDocumentationLink, "TraceQualityDocumentationLink should match")
```

Comment:
did you mean NotEmpty? A string field can never be nil.
```suggestion
	assert.NotEmpty(t, data.TraceQualityDocumentationLink, "TraceQualityDocumentationLink should match")
```
```suggestion
	assert.NotNil(t, data.TraceQualityDocumentationLink, "TraceQualityDocumentationLink should match")
```